cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -161,19 +174,100 @@ const requestCertificate = async function (
   return reqCertResponse.CertificateArn;
 };
 
+const updateHostedZoneRecords = async function (
+  action,
+  options,
+  envName,
+  appName,
+  domainName,
+  envRoute53,
+  appRoute53,
+  envHostedZoneId
+) {
+  const promises = [];
+  for (const option of options) {
+    switch (option.DomainName) {
+      case `${envName}.${appName}.${domainName}`:
+        promises.push(validateDomain(
```

Comment:
nit: can we keep the order of the parameters consistent across functions, or as an alternative we can use objects as inputs instead. For example:
```javascript
promises.push(validateDomain({
   client: envRoute53,
   record: option.ResourceRecord,
   ...
   domainName: "",
})
```
It was kinda difficult to follow which value mapped to which parameter with so many strings involved 😄